dask_geopandas/backends.py

```diff
@@ -13,13 +13,17 @@
 import geopandas
 from geopandas.array import GeometryArray, GeometryDtype, from_shapely
 
-DASK_2021_06 = str(dask.__version__) > LooseVersion("2021.05.0")
+DASK_2021_05_1 = str(dask.__version__) >= LooseVersion("2021.05.1")
+DASK_2021_06_0 = str(dask.__version__) > LooseVersion("2021.05.1")
 
-if DASK_2021_06:
-    from dask.dataframe.utils import make_meta
+if DASK_2021_06_0:
+    from dask.dataframe.dispatch import make_meta_dispatch
+    from dask.dataframe.backends import _nonempty_index, meta_nonempty_dataframe
+elif DASK_2021_05_1:
```

Comment:
This branch is just to cover the `dask=2021.05.1` release. I'm not sure if you'd prefer to keep this extra code branch, or just say "dask 2021.05.1 was a bad release, don't use it"

Reply:
I would probably go with "don't use 2021.05.1".

Reply:
Depending on when the next dask release will drop, we may need to add a pin to CI envs, which will now load 2021.05.1 and fail if we remove this condition.

Reply:
> I would probably go with "don't use 2021.05.1"

+1 I'll update this PR accordingly

We're planning to release tomorrow (xref https://github.com/dask/community/issues/162). We can also just wait to merge this PR until `dask=2021.06.0` is out, which should help keep CI passing

Reply:
> We can also just wait to merge this PR until dask=2021.06.0 is out, which should help keep CI passing

Sounds good to me. I'll keep an eye on the release and merge after. Thanks!

Reply:
Alright, just updated this PR to simplify things a bit. I can ping this PR again once the next release is out 👍

Reply:
> I would probably go with "don't use 2021.05.1".

Probably worth adding a version constraint to `setup.py`

https://github.com/geopandas/dask-geopandas/blob/dd21f7c7479b55e8af673757af3e79b1e87277f6/setup.py#L6

Reply:
Just added `!=2021.05.1` to the version requirements in `setup.py`